### PR TITLE
[WIP]Add doc of load PyTorch

### PR DESCRIPTION
**Pull request by ManfeiBai**
Add load PyTorch in doc

### CODE REVIEW

File: docs/docs/APIGuide/PipelineAPI/inference.md [hunk 1]
```diff
@@ -162,6 +163,27 @@ model.load_openvino(modelPath, weightPath)
 * `modelPath`: String. Path of pre-trained OpenVINO model.
 * `weightPath`: String. Path of pre-trained OpenVINO model weight.
 
+### **Load pre-trained PyTorch model**
+Load Pytorch model with corresponding `loadPyTorch` methods (`loadPyTorch` for Java and Python, `doLoadPyTorch` for Scala).
```
Comment (qiuxin2012) [inline docs/docs/APIGuide/PipelineAPI/inference.md:167]: Please add note, user should save torch model using `zoo_pickle_module`, like
```
from zoo.pipeline.api.torch import zoo_pickle_module
model = models.resnet18(pretrained = True)
torch.save(model, "$modelPath", pickle_module=zoo_pickle_module)
```

Comment (ManfeiBai) [reply docs/docs/APIGuide/PipelineAPI/inference.md]: Done~

File: docs/docs/APIGuide/PipelineAPI/inference.md [hunk 2]
```diff
@@ -162,6 +162,46 @@ model.load_openvino(modelPath, weightPath)
...
+torch.save(model, "$modelPath", pickle_module=zoo_pickle_module)
+```
+
+Then load PyTorch model:
+
+**Python**
+
+```python
+model = InferenceModel()
+model.load_pytorch(modelPath, weightPath)
```
Comment (qiuxin2012) [inline docs/docs/APIGuide/PipelineAPI/inference.md:182]: torch doesn't need weight path

Comment (ManfeiBai) [reply docs/docs/APIGuide/PipelineAPI/inference.md:182]: Thanks! Fix soon~

**Pull request closed without merging**